static/app/views/profiling/content.tsx

```diff
@@ -98,6 +100,8 @@ function ProfilingContent({location, selection}: ProfilingContentProps) {
                   isLoading={requestState === 'loading'}
                 />
                 <ProfilingTable
+                  organization={organization}
```

Comment:
Why not just have ProfilingTable use useProjects and useOrganization instead of passing it down?